config/patches/pkg-config-lite/pkg-config-lite-0.28-1.config.guess.patch

```diff
@@ -0,0 +1,992 @@
+--- config.guess_OLD	2015-10-30 21:16:33.910969812 +0000
++++ config.guess	2015-10-30 21:16:39.771905250 +0000
+@@ -1,14 +1,12 @@
+ #! /bin/sh
+ # Attempt to guess a canonical system name.
+-#   Copyright (C) 1992, 1993, 1994, 1995, 1996, 1997, 1998, 1999,
+-#   2000, 2001, 2002, 2003, 2004, 2005, 2006, 2007, 2008, 2009, 2010
+-#   Free Software Foundation, Inc.
++#   Copyright 1992-2015 Free Software Foundation, Inc.
+ 
+-timestamp='2009-12-30'
++timestamp='2015-10-21'
+ 
+ # This file is free software; you can redistribute it and/or modify it
+ # under the terms of the GNU General Public License as published by
+-# the Free Software Foundation; either version 2 of the License, or
++# the Free Software Foundation; either version 3 of the License, or
```

Comment:
The interesting exception in this case is a few lines below, see line 30, copied here:

```
# As a special exception to the GNU General Public License, if you
# distribute this file as part of a program that contains a
# configuration script generated by Autoconf, you may include it under
# the same distribution terms that you use for the rest of that
# program.  This Exception is an additional permission under section 7
# of the GNU General Public License, version 3 ("GPLv3").
```

See also the interpretation here: https://lists.debian.org/debian-devel-announce/2013/05/msg00017.html

So pkg-config-lite, which has GPLv2 explicitly in its COPYING file, could distribute this file under the terms of GPLv2 because it has an autoconf configuration script.

But, there's a fine line wherein it is not pkg-config-lite that is distributing this file. It is Chef Software, and neither omnibus-software or omnibus have any need for an autoconf script. So, we should be absolutely clear what the companies position is here. There's an argument to be made (and has been made in media cases), that making a work available [online] is distribution, even if it isn't included in the package that we ship. That's an academic exercise for lawyers, as nobody is really going to get sued over where they put config.guess on the internet.

But @lamont-granquist is exactly right. GPLv3 FUD scares people, and generally we've felt that we don't have time for that. Although I'm not sure if his argument was for ripping out gdbm or excluding config.guess as presented.